geomstats/geometry/special_orthogonal.py

```diff
@@ -101,29 +101,17 @@ def _is_in_lie_algebra(self, tangent_vec, atol=ATOL):
         return self.lie_algebra.belongs(tangent_vec, atol=atol)
 
     @classmethod
-    def _to_lie_algebra(cls, vec):
-        """Project vector onto skew-symmetric matrices.
-
-        Parameters
-        ----------
-        vec : array-like, shape=[..., n, n]
-            Vector.
-
-        Returns
-        -------
-        tangent_vec : array-like, shape=[..., n, n]
-            Tangent vector at base point.
-        """
-        return cls.to_skew_symmetric(vec)
+    def _to_lie_algebra(cls, tangent_vec):
```

Comment:
There seems to be a GitHub issue: the docstrings of this file seem to have disappeared in your GitHub branch. Maybe rebase?